app/models.py

```diff
@@ -57,9 +59,16 @@ class Post(db.Model):
     job_type = db.Column(db.String(100), nullable=True)
     description = db.Column(db.Text, nullable=True)
     salary = db.Column(db.Integer, nullable=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False, default = 1) # lowercase becuase referencing column NOT class
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False, default = 1) # lowercase because referencing column NOT class
     #user_id logic needs to be fixed
 
     def __repr__(self):
         return f"Post('{self.title}', '{self.date_posted})"
-    
\ No newline at end of file
+
+class Account(db.Model):
```

Comment:
nice addition of account class. I think just the one relationship with user is a good call.